front/lib/api/assistant/conversation.ts

```diff
@@ -645,6 +649,22 @@ export async function* postUserMessage(
   ]);
   const agentConfigurations = results[0];
 
+  const whiteListedProviders = owner.whiteListedProviders ?? MODEL_PROVIDER_IDS;
+  for (const agentConfig of agentConfigurations) {
+    if (!whiteListedProviders.includes(agentConfig.model.providerId)) {
+      yield {
+        type: "agent_disabled_error",
+        created: Date.now(),
+        configurationId: agentConfig.sId,
+        error: {
+          code: "provider_disabled",
+          message: `The model provider for agent configuration ${agentConfig.sId} is currently disabled.`,
```

Comment:
Let's use the assistant name and say that it was disabled by the workspace admin:

Assistant {assistant name} is based on a model that was disabled by your workspace admin. Please edit the assistant to use another model (advanced settings in the Instructions panel).

Reply:
👍🏼

Reply:
@spolu also added the check in `editUserMessage`